content/docs/getting-started/installation/kubernetes/powerflex/csmoperator/_index.md

```diff
@@ -157,6 +157,7 @@ run `/opt/emc/scaleio/sdc/bin/drv_cfg --add_mdm --ip 10.xx.xx.xx.xx,10.xx.xx.xx`
    | X_CSI_DEBUG | To enable debug mode | No | true |
    | X_CSI_ALLOW_RWO_MULTI_POD_ACCESS | Setting allowRWOMultiPodAccess to "true" will allow multiple pods on the same node to access the same RWO volume. This behavior conflicts with the CSI specification version 1.3. NodePublishVolume description that requires an error to be returned in this case. However, some other CSI drivers support this behavior and some customers desire this behavior. Customers use this option at their own risk. | No | false |
    | INTERFACE_NAMES | A mapping of node names to interface names. Only necessary when SDC is disabled. | No | none |
+   | CSI_LOG_LEVEL | Controls the log level for messages printed to the driver logs. Allowable values per logrus are [INFO, DEBUG, TRACE, WARN, ERROR, FATAL, and PANIC](https://github.com/sirupsen/logrus/blob/master/logrus.go#L25). | No | INFO |
```

Comment:
I would rather not mention our underlying logger implementation in case it changes at some point.  Let's just keep it as simple as possible for our users.